ci/prow_setup.md

```diff
@@ -12,13 +12,10 @@
    https://github.com/knative/test-infra/issues/1082.
 
 1. Ensure the GCP projects listed in
```

Comment:
I'm making way more changes to this file, including the fixes you have here, in https://github.com/knative/test-infra/pull/1441 you can take this file out of this PR.

Reply:
Removed.